crates/proof-of-sql/src/proof_primitive/dory/public_parameters.rs

```diff
@@ -21,14 +25,19 @@ pub struct PublicParameters {
 }
 
 impl PublicParameters {
+    /// Generate cryptographically secure random public parameters.
+    pub fn crypto_rand<R: CryptoRng + Rng + ?Sized>(max_nu: usize, rng: &mut R) -> Self {
```

Comment:
This `CryptoRng` enforces cryptographic security, so there isn't a need to name it differently.
```suggestion
    #[cfg(not(any(test, feature = "test")))]
    /// Generate random public parameters.
    pub fn rand<R: CryptoRng + Rng + ?Sized>(max_nu: usize, rng: &mut R) -> Self {
```

If you really don't like this, I would name this `rand` and the test version `test_rand`. But, that is a breaking change.

Reply:
I tried that originally and it produced a rather large diff, this seemed like the least impactful way to add this in

Reply:
You tried the suggestion? Or renaming it to `test_rand`. That code I suggested should be the only change needed.